### ISSUE TITLE
VM Migration - warnings in log

### ISSUE THREAD

**Issue by feldsam** [OpenNebula/one#1690]
# Bug Report
## Version of OpenNebula

- [ ] 5.2.2
- [ ] 5.4.0
- [ ] 5.4.1
- [ ] 5.4.2
- [X] 5.4.6
- [ ] Development build

## Component

- [ ] Authorization (LDAP, x509 certs...)
- [ ] Command Line Interface (CLI)
- [ ] Contextualization
- [ ] Documentation
- [ ] Federation and HA
- [ ] Host, Clusters and Monitorization
- [X] KVM
- [ ] Networking
- [ ] Orchestration (OpenNebula Flow)
- [ ] Packages
- [ ] Scheduler
- [ ] Storage & Images
- [ ] Sunstone
- [ ] Upgrades
- [ ] User, Groups, VDCs and ACL
- [ ] vCenter

## Description
I see warnings in log when migrating VM

    Sat Jan 27 19:14:45 2018 [Z0][VM][I]: New LCM state is RUNNING
    Sat Jan 27 19:15:59 2018 [Z0][VM][I]: New LCM state is SAVE_MIGRATE
    Sat Jan 27 19:16:03 2018 [Z0][VMM][I]: /var/tmp/one/vmm/kvm/save: line 58: warning: command substitution: ignored null byte in input
    Sat Jan 27 19:16:03 2018 [Z0][VMM][I]: ExitCode: 0
    Sat Jan 27 19:16:03 2018 [Z0][VMM][I]: Successfully execute virtualization driver operation: save.
    Sat Jan 27 19:16:04 2018 [Z0][VMM][I]: ExitCode: 0
    Sat Jan 27 19:16:04 2018 [Z0][VMM][I]: Successfully execute network driver operation: clean.
    Sat Jan 27 19:16:04 2018 [Z0][VM][I]: New LCM state is PROLOG_MIGRATE
    Sat Jan 27 19:16:04 2018 [Z0][VM][I]: New LCM state is BOOT_MIGRATE
    Sat Jan 27 19:16:05 2018 [Z0][VMM][I]: Successfully execute transfer manager driver operation: tm_context.
    Sat Jan 27 19:16:05 2018 [Z0][VMM][I]: ExitCode: 0
    Sat Jan 27 19:16:05 2018 [Z0][VMM][I]: Successfully execute network driver operation: pre.
    Sat Jan 27 19:16:08 2018 [Z0][VMM][I]: /var/tmp/one/vmm/kvm/restore: line 43: warning: command substitution: ignored null byte in input
    Sat Jan 27 19:16:08 2018 [Z0][VMM][I]: ExitCode: 0
    Sat Jan 27 19:16:08 2018 [Z0][VMM][I]: Successfully execute virtualization driver operation: restore.
    Sat Jan 27 19:16:09 2018 [Z0][VMM][I]: ExitCode: 0
    Sat Jan 27 19:16:09 2018 [Z0][VMM][I]: Successfully execute network driver operation: post.
    Sat Jan 27 19:16:09 2018 [Z0][VM][I]: New LCM state is RUNNING

**Comment by feldsam** [comment]
This also happens on stop operation

    Thu Apr 19 13:39:41 2018 [Z0][VM][I]: New LCM state is RUNNING
    Thu Apr 19 13:45:27 2018 [Z0][VM][I]: New LCM state is SAVE_STOP
    Thu Apr 19 13:45:29 2018 [Z0][VMM][I]: /var/tmp/one/vmm/kvm/save: line 58: warning: command substitution: ignored null byte in input

**Comment by feldsam** [comment]
Hello, this is still there, even on 5.6.1.

```
Fri Nov 23 15:55:31 2018 [Z0][VM][I]: New LCM state is SAVE_MIGRATE
Fri Nov 23 15:55:58 2018 [Z0][VMM][I]: /var/tmp/one/vmm/kvm/save: line 58: warning: command substitution: ignored null byte in input
Fri Nov 23 15:55:58 2018 [Z0][VMM][I]: ExitCode: 0
Fri Nov 23 15:55:58 2018 [Z0][VMM][I]: Successfully execute virtualization driver operation: save.
Fri Nov 23 15:55:59 2018 [Z0][VMM][I]: ExitCode: 0
Fri Nov 23 15:55:59 2018 [Z0][VMM][I]: Successfully execute network driver operation: clean.
Fri Nov 23 15:55:59 2018 [Z0][VM][I]: New LCM state is PROLOG_MIGRATE
Fri Nov 23 15:55:59 2018 [Z0][VM][I]: New LCM state is BOOT_MIGRATE
Fri Nov 23 15:55:59 2018 [Z0][VM][I]: Virtual Machine has no context
Fri Nov 23 15:56:00 2018 [Z0][VMM][I]: pre: Executed "sudo brctl addbr onebr18".
Fri Nov 23 15:56:00 2018 [Z0][VMM][I]: pre: Executed "sudo ip link set onebr18 up".
Fri Nov 23 15:56:00 2018 [Z0][VMM][I]: pre: Executed "sudo ip link add link team0 name team0.218 mtu 1500 type vlan id 218 gvrp on ".
Fri Nov 23 15:56:00 2018 [Z0][VMM][I]: pre: Executed "sudo ip link set team0.218 up".
Fri Nov 23 15:56:00 2018 [Z0][VMM][I]: pre: Executed "sudo brctl addif onebr18 team0.218".
Fri Nov 23 15:56:00 2018 [Z0][VMM][I]: ExitCode: 0
Fri Nov 23 15:56:00 2018 [Z0][VMM][I]: Successfully execute network driver operation: pre.
Fri Nov 23 15:56:24 2018 [Z0][LCM][I]: VM reported RUNNING by the drivers
Fri Nov 23 15:56:24 2018 [Z0][VM][I]: New LCM state is RUNNING
Fri Nov 23 15:56:27 2018 [Z0][VMM][I]: /var/tmp/one/vmm/kvm/restore: line 43: warning: command substitution: ignored null byte in input
Fri Nov 23 15:56:27 2018 [Z0][VMM][I]: ExitCode: 0
Fri Nov 23 15:56:27 2018 [Z0][VMM][I]: Successfully execute virtualization driver operation: restore.
Fri Nov 23 15:56:28 2018 [Z0][VMM][I]: ExitCode: 0
Fri Nov 23 15:56:28 2018 [Z0][VMM][I]: Successfully execute network driver operation: post.
Fri Nov 23 15:56:28 2018 [Z0][LCM][E]: deploy_success_action, VM in a wrong state
```

**Comment by rsmontero** [comment]
Fixed in master and one-5.10 for 5.10.2
